src/awkward/highlevel.py

```diff
@@ -1404,9 +1454,14 @@ def show(
             limit_cols (int): Maximum number of columns (characters wide).
             type (bool): If True, print the type as well. (Doesn't count toward number
                 of rows/lines limit.)
+            named_axis (bool): If True, print the named axis as well. (Doesn't count toward number
+                of rows/lines limit.)
+            nbytes (bool): If True, print the number of bytes as well. (Doesn't count toward number
+                of rows/lines limit.)
+            nbytes (bool): If True, print the backend of the array as well. (Doesn't count toward number
+                of rows/lines limit.)
```

Comment:
Eventually, someday, `all: bool` would set all of these to `True`.

Reply:
Also, the argument is spelled `named_axis` here but is presented as `axes` in the text output.

Yet one more thing: these are all boolean arguments, but they're allowed to be positional (they're before the `*`). As an API design, it's safer to force users to type the keyword names of contiguous boolean arguments.

Reply:
> Eventually, someday, all: bool would set all of these to True.

"Someday" will be (hopefully) tomorrow when I add `all: bool` :+1:

> Also, the argument is spelled named_axis here but is presented as axes in the text output.

Good point, I'll change `axes:` to `named axis:`.

> Yet one more thing: these are all boolean arguments, but they're allowed to be positional (they're before the *). As an API design, it's safer to force users to type the keyword names of contiguous boolean arguments.

Sounds good, will change that :+1:

Reply:
all comments addressed